dataflow/src/main/java/org/checkerframework/dataflow/util/DataflowTester.java

```diff
@@ -0,0 +1,65 @@
+package org.checkerframework.dataflow.util;
+
+import org.checkerframework.dataflow.analysis.Analysis;
+import org.checkerframework.dataflow.cfg.visualize.CFGVisualizeLauncher;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * A utils performs dataflow tests or playgrounds.
+ *
+ * <p>A DataflowTester receive the {@link Analysis} instance from specific dataflow test/playground,
+ * e.g., LiveVariable, then generate CFG by using this instance. PerformTest reads from the input
+ * file inputFile and then generates an output file of CFG, which will be compared with an
+ * expectation output file under the working directory; while performPlayground generates the dot of
+ * CFG.
+ */
+public class DataflowTester {
```

Comment:
It's a little bit strange to name it `Tester` while it also has a function for playground.